<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentDetailDlg</class>
 <widget class="QDialog" name="PaymentDetailDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>324</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>551</width>
    <height>324</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="TitleBarWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="backgroundWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>284</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#backgroundWidget{
	border-image: url(:/paymentImage/payDetail_background.png);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:14px;
font-family:&quot;微软雅黑&quot;;
color:#999999;</string>
          </property>
          <property name="text">
           <string>整单金额：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="simpleAmtValueLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ffffff;
font-family:&quot;微软雅黑&quot;;
font-size:14px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>￥0.00</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>15</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:14px;
font-family:&quot;微软雅黑&quot;;
color:#999999;</string>
          </property>
          <property name="text">
           <string>已收金额：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="PaidAmtValueLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ffffff;
font-family:&quot;微软雅黑&quot;;
font-size:14px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>￥0.00</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:14px;
font-family:&quot;微软雅黑&quot;;
color:#999999;</string>
          </property>
          <property name="text">
           <string>待收金额：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="receivedAmtValueLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ffffff;
font-family:&quot;微软雅黑&quot;;
font-size:14px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>￥0.00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cashRefundBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#cashRefundBtn {
	border-image: url(:/statementImage/cashRefund_nomal.png);
	color: rgb(255, 255, 255);
	border:none;
	font-size:14px;
}

QPushButton#cashRefundBtn:pressed {
	border-image: url(:statementImage/cashRefund_press.png);
}</string>
          </property>
          <property name="text">
           <string>现金退款</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PaymentDetailWidget" name="payDetailsWidget" native="true">
        <property name="minimumSize">
         <size>
          <width>512</width>
          <height>190</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>512</width>
          <height>190</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:16px;
font-family:&quot;微软雅黑&quot;;
color:#999999;</string>
          </property>
          <property name="text">
           <string>共 </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalPayDetailsLabel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ffffff;
font-family:&quot;微软雅黑&quot;;
font-size:16px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:16px;
font-family:&quot;微软雅黑&quot;;
color:#999999;</string>
          </property>
          <property name="text">
           <string> 笔</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleBarWidget</class>
   <extends>QWidget</extends>
   <header location="global">TitleBarWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PaymentDetailWidget</class>
   <extends>QWidget</extends>
   <header>PaymentDetailWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
